experimenter/experimenter/features/manifests/apps.yaml

```diff
@@ -29,7 +29,9 @@ firefox_ios:
   release_discovery:
     version_file:
       type: "plist"
-      path: "Client/Info.plist"
+      path:
```

Comment:
This is an example where we could've made it a new field called 'paths' that only takes a list rather than folding two types into one field?  We control both sides so we don't really need to reason about backwards compatibility, just musing.

Reply:
We could, but then the pydantic schema needs code to validate that one of `paths` or `path` is specified, which is more complicated IMO